ecs-cli/modules/commands/cluster/cluster_command.go

```diff
@@ -122,6 +122,11 @@ func clusterUpFlags() []cli.Flag {
 			Name:  flags.VpcIdFlag,
 			Usage: "[Optional] Specifies the ID of an existing VPC in which to launch your container instances. If you specify a VPC ID, you must specify a list of existing subnets in that VPC with the --subnets option. If you do not specify a VPC ID, a new VPC is created with two subnets.",
 		},
+		cli.StringSliceFlag{
+			Name:  flags.UserDataFlag,
+			Usage: "[Optional] Specify additional User Data to be used with your EC2 instances; the CLI automatically creates User Data which directs your instances to join your ECS Cluster. User Data can be shell scripts or cloud-init directives, consult the EC2 documentation for more info. Multiple files can be specified, the ECS CLI will pack them all into a MIME Multipart Archive.",
```

Comment:
This is another really long line.  Can we shorten it?

Reply:
Suggested reword:
* Specifies additional User Data for your EC2 instances. Files can be shell scripts or cloud-init directives and are packaged into a MIME Multipart Archive along with CLI-provided User Data which directs instances to join your cluster.